### DISCUSSION TITLE
Getting array_map() error when opening versions of a data object

### DISCUSSION THREAD

**Discussion by utkarsh-sharma-511** [pimcore/pimcore#10454] [Q&A / Support]
array_map(): Argument #1 ($callback) must be a valid callback, array must have exactly two members

[](url)
getting this error when trying to view version of an Object.
I have attached image for the same.
![Screenshot_3](https://user-images.githubusercontent.com/88316749/135268407-94c45ec0-53ca-4039-814f-1cab0c7a4703.png)

Can any one please help?

**Comment by dvesh3** [comment]
already fixed on 10.1.3. please see https://github.com/pimcore/pimcore/pull/10224
